tc-tests-utils.sh

```diff
@@ -415,6 +424,10 @@ verify_bazel_rebuild()
     exit 1
   fi;
 
+  mkdir -p ${TASKCLUSTER_ARTIFACTS} || true
```

Comment:
This helps us debug more easily in case of spurious rebuild: task will fail, and thus the produced log won't be uploaded since package.sh won't be called.